ui/js/actions/app.js

```diff
@@ -219,3 +219,15 @@ export function doChangeVolume(volume) {
     });
   };
 }
+
+export function doOpenOverlayMedia() {
```

Comment:
I suspect this code can be written without this action. We can know whether or not it should be open solely from existing app state.